### ISSUE TITLE
Yolov4-tiny-416.tflite custom trained model not working in SSD scene

### ISSUE THREAD

**Issue by syedali1621** [asus4/tf-lite-unity-sample#89]
- Unity Version: [e.g. Unity 2020.1.6]

I have trained my own dataset using Yolov4tiny darknet and converted the .weight file into .pb file then further converted .pb to .tflite i am trying to run my .tflite file on unity ssd scene but facing below errors. I have changed the label file path to my label file and also changed .tffile path. 

Exception: TensorFlowLite operation failed.
TensorFlowLite.Interpreter.ThrowIfError (TensorFlowLite.Interpreter+Status status) (at Packages/com.github.asus4.tflite/Runtime/Interpreter.cs:181)
TensorFlowLite.Interpreter.SetInputTensorData (System.Int32 inputTensorIndex, System.Array inputTensorData) (at Packages/com.github.asus4.tflite/Runtime/Interpreter.cs:97)
TensorFlowLite.SSD.Invoke (UnityEngine.Texture inputTex) (at Assets/Samples/SSD/SSD.cs:32)
SsdSample.Update () (at Assets/Samples/SSD/SsdSample.cs:58)

**Comment by asus4** [comment]
Need more detail about this issue. Please refer #7 and confirm the model input/output and type/size are correct.

**Comment by syedali1621** [comment]
![416](https://user-images.githubusercontent.com/14038234/104678070-fb86ea80-572d-11eb-9631-754b541c1148.PNG)
Well my model properties are different do i need same properties to run my model in unity SSD scene?
i trained using Alexab darknet then convert into .tflite i can change input size to 300*300 but i don't know how to change other properties. How you trained coco_ssd_mobilenet_quant.tflite model which repository you have used i think i need to train with same repository to get same model properties?

**Comment by asus4** [comment]
This repo contains samples of only pre-traind models in the TensorFlow/MediaPipe official repo. If you wanted to use the same SSD model, please refer [the link](https://www.tensorflow.org/lite/models/object_detection/overview#model_customization)

You can modify the code to adapt your model.
The input/output format should be like this:
- float[416,416,3] input0
- float[4] output0: guess it's a bounding box?
- float output1: score?

**Comment by syedali1621** [comment]
Thank you for your answer could you please explain which script do I need to modify?

**Comment by syedali1621** [comment]
![300300](https://user-images.githubusercontent.com/14038234/104899267-fb048300-59bd-11eb-81cc-d9344364f5ab.PNG)
I have trained another model with same 300*300 image size. converted .pb file into tflite but still model is not running in unity . Could you please tell me which script do I need to modify to run it in unity ?

**Comment by bertafmorena** [comment]
> 

Could you pls specify which scripts do I have to modify to use my custom model with input_type = float32[1,640,640,3]
